datacommons/R/zzz.R

```diff
@@ -23,6 +23,7 @@ dc <- NULL
   # setting delay_load delays loading the module until it is first used
   # thus, user can first set python version in R after loading in this
   # R API Client package
-  system("pip install --upgrade --user --quiet git+https://github.com/datacommonsorg/api-python.git@v1.0.1")
-  dc <<- reticulate::import("datacommons", delay_load = FALSE)
+  system("pip3 install --upgrade --user --quiet git+https://github.com/datacommonsorg/api-python.git@v1.0.1")
```

Comment:
should just use datacommons here?

Reply:
Would need to spend some time to make R library compatible without pandas